### ISSUE TITLE
Bug in CosineAnnealingWarmRestarts (T_mul instead of T_mult)

### ISSUE THREAD

**Issue by ldutu** [pytorch/pytorch#22542]
## 🐛 Bug

The bug appears in PyTorch 1.1 in file lr_scheduler.py (line 686) for LR scheduler "CosineAnnealingWarmRestarts", where the parameter name is *T_mult* but in line 686 the name *T_mul* is used instead (python interpreter does not know variable *T_mul*).

## To Reproduce

The bug can be easily reproduced if the *T_mult* parameter of CosineAnnealingWarmRestart scheduler is not set properly (i.e. is < 1 or not an instance of int), causing Python to raise the error at line 686 (where *T_mul* is used instead of *T_mult*).

The following line will raise the error and cause the bug:
`scheduler = optim.lr_scheduler.CosineAnnealingWarmRestarts(optimizer, T_0=100, T_mult=2.0)`

because 2.0 is not an instance of int.

## Expected behavior

Line 686 of lr_scheduler.py should be modified from
`raise ValueError("Expected integer T_mul >= 1, but got {}".format(T_mul))`
into 
`raise ValueError("Expected integer T_mult >= 1, but got {}".format(T_mult))`

## Environment

- PyTorch Version (e.g., 1.0): 1.1
 - OS (e.g., Linux): Linux
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.7
 - CUDA/cuDNN version: 10
 - GPU models and configuration: 1080 Ti
 - Any other relevant information: N/A

**Comment by mrshenli** [comment]
Hi @ldutu, thanks for reporting! Since you have found the solution, would you like to submit a PR for it? We will be happy to review and land.

**Comment by ldutu** [comment]
Hi @mrshenli , checking the latest version of the file, I found that this was already fixed by @rtqichen in commit 57948414acf87c846bf88cf5f194282b8f47c423 . Thanks!
